package.json

```diff
@@ -1,24 +1,26 @@
 {
-  "name": "onboarding-serverless-todo",
-  "version": "1.0.0",
-  "description": "This repo is meant to be used for PR analysis of people learning serverless framework",
-  "main": "index.js",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "say": "cowsay \"I am here!\""
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/nielsmijnster/onboarding-serverless-todo.git"
+  "devDependencies": {
+    "@babel/preset-typescript": "^7.18.6",
+    "@types/jest": "^28.1.8",
+    "@types/node": "^16.0.0",
+    "gts": "^4.0.0",
+    "jest": "^28.1.3",
+    "ts-jest": "^28.0.8",
+    "typescript": "~4.7.0"
   },
-  "keywords": [],
-  "author": "",
-  "license": "ISC",
-  "bugs": {
-    "url": "https://github.com/nielsmijnster/onboarding-serverless-todo/issues"
+  "scripts": {
+    "lint": "gts lint",
+    "clean": "gts clean",
+    "compile": "tsc",
+    "fix": "gts fix",
+    "prepare": "npm run compile",
+    "pretest": "npm run compile",
+    "posttest": "npm run lint",
+    "slug": "slug.ts",
```

Comment:
Is this working correctly? Outputting 10 random sentences when called?

Reply:
<img width="628" alt="image" src="https://user-images.githubusercontent.com/108457192/187507687-25ab69e6-73c1-44d4-9ff5-3bddd670bc1d.png">

Reply:
so I think no, because I don't use the slug but just request the file to run

Reply:
changed it now, while running npm run slug, this ts-node I use because otherwise I have to keep doing tsc slug.ts and then node slug.js.
<img width="587" alt="image" src="https://user-images.githubusercontent.com/108457192/187508305-466f9bf8-fcb1-4412-86e3-43a271ac3e49.png">

Reply:
It's ok!

Reply:
Regarding to having to call tsc before run is normal. One step will build and the other run. So normally you use the script "build" to prepare and "slug" to actually run the program. That's why before running a package you should always run "npm install", because then it will will install all dependencies and then run the "build" script os your package to prepare anything you need for your package to be ready to run

Reply:
aha clear now, first build, and then the script itself thanks